### ISSUE TITLE
Visualization issue with event title in event management area

### ISSUE THREAD

**Issue by markpolyak** [indico/indico#6673]
**Describe the bug**
Event title in the event management area is not displayed properly. The text overlaps and is somehow cropped. On the screenshot you can see that the conference title is long and doesn't fit on one line. The second line of the title is displayed too close to the first line and they overlap, see arrow No 1 on the screenshot below. Arrow No 2 points to a Cyrillic letter `щ` in the title of the event, which is displayed as `ш`, which is a different letter (without the tick at the bottom right). You can see at the arrow No 3 that actual event title has letter `щ`, not `ш`. I assume there is an issue with the font and/or styles here. I've also checked on Ubuntu 24.04 in case it is a font issue on MacOS. The issue with overlapping title and wrong letter is present there as well, see the second screenshot.

I'm using MacOS Ventura, the issue is present in Safari 17.6 and Chrome 131.0.6778.109.

**To Reproduce**
1. Go to event management area
2. Edit event title to make it long enough to not fit in one line
3. Add cyrillic letter `щ` to the title of the event

**Expected behavior**
Event title should be displayed properly

**Screenshots**
MacOS:
![2024-12-13_21-42-38](https://github.com/user-attachments/assets/c989505c-bcf1-4c54-aa75-f581fd0a83d3)
Ubuntu:
![2024-12-14_03-59-53](https://github.com/user-attachments/assets/4fca1e71-6ed5-4072-a69d-af283d6269b9)

**Additional context**
Oddly enough Cyrillic letter `ц`, which also has a tick at the bottom right, is not cropped. So the issue of overlapping text and the issue of a wrong Cyrillic character being displayed might not be connected with each other.

**Comment by markpolyak** [comment]
Okay, there are indeed two separate issues here. 

I was able to track the issue with Cyrrilic letter `щ` down to the file `indico/web/static/fonts/roboto/Roboto-Light-webfont.woff`. This particular letter is ambigious in this particular font file. The ttf file with that font locasted in the same directory has the same issue, so I assume that all static files for this font are from the same source and have the same issue. Metadata shows that it is `Roboto Light. Version 1.00000; 2011`. 

The Roboto-Light font [currently available](https://fonts.google.com/specimen/Roboto) from Google Fonts is `Version 2.137; 2017` and **it doesn't have the issue with letter `щ`**. So my suggestion would be to update the static font files for Roboto-Light to the latest available version of that font. Going further it might make sense to update other fonts located in the `indico/web/static/fonts` directory as well, since they seem to be pretty old.

**Comment by markpolyak** [comment]
Here is a comparison of currently used erroneous font and a newer version of that font I suggest should be used instead.

Glyphs of Roboto-Light font currently used in Indico (v 1.0 from 2011). Uppercase Cyrillic letters `Ш` and `Щ` are different (i.e. correct), while lowercase `ш` and `щ` are identical:
![2024-12-19_04-16-37](https://github.com/user-attachments/assets/a4ff0c80-9d4d-4f68-b134-8f8a0f5a74d3)


Glyphs of Roboto-Light font available from Google Fonts (v 2.137 from 2017). The issue with lowercase letters is fixed:
![2024-12-19_04-40-55](https://github.com/user-attachments/assets/b89f6a75-7641-43da-b904-bd4a7b07f24b)

**Comment by ThiefMaster** [comment]
So the fix is as simple as updating the font on our side? :)

**Comment by markpolyak** [comment]
Yes, that would fix the issue with Cyrillic `щ`. The issue of overlapping text has a different origin though.

**Comment by markpolyak** [comment]
The issue with overlapping text seems to be caused by the `.title` style [missing](https://github.com/indico/indico/blob/074f468566075e3412e3888f56b5f7a85a20ceda/indico/web/client/styles/base/_pages.scss#L153) the `line-height` property. That's why this property gets inherited from the indico-sui-theme `body` tag [style](https://github.com/indico/indico-sui-theme/blob/45ca4ab8a9a28c9d6488d57d7b7d3b308f5d9b31/semantic.css#L80):

```
body {
  ...
  line-height: 1.4285em;
  ...
}
```

According to Mozilla Web CSS docs, it is recommended to use a unitless value for the `line-height` property to avoid unexpected results caused by poor inheritance behaviour. They even have an [example](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#prefer_unitless_numbers_for_line-height_values) that illustrates our exact issue here.

It is easy to add a `line-height` property to the `.title` style in `indico/web/client/styles/base/_pages.scss`. Alternatively it might make sense to update the indico-sui-theme to use a unitless `line-height` property for the `body` tag, although I'm not competent enough to estimate the consequences and feasibility of such an update as it will probably break other things elsewhere. I've tested both solutions on a couple of event management pages and they look good in Safari (`line-height: 1.1;`).
